recipes/rnabob/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: rnabob
+  version: "2.2.1"
+
+source:
+  url: http://eddylab.org/software/rnabob/rnabob.tar.gz
```

Comment:
Oh, I didn't find it through http://eddylab.org/software/rnabob/, but from here: http://eddylab.org/software.html

I guess that means that the link I used points to http://eddylab.org/software/rnabob/rnabob-2.2.1.tar.gz.

Should I modify it in the meta.yaml?